ports/kf5auth/vcpkg.json

```
@@ -0,0 +1,12 @@
+{
+  "name": "kf5auth",
+  "version-string": "5.75.0",
```

```suggestion
  "version": "5.75.0",
```
We prefer to use more proper name to describe this field. Please see https://github.com/microsoft/vcpkg/blob/master/docs/specifications/versioning.md#22-package-versions

---

Yes, I am actually aware – I was researching wether we're compliant with SymVer, I'll update accordingly.